src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CompilerExtensionsResources.resx

```diff
@@ -279,4 +279,67 @@
   <data name="New_line_preferences" xml:space="preserve">
     <value>New line preferences</value>
   </data>
+  <data name="camelCase" xml:space="preserve">
+    <value>camelCase</value>
```

Comment:
This shouldn't be localized right? I'd add a "Locked" note for that.

Reply:
good idea thanks!